lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -604,9 +605,22 @@ func asRustBackedSlice(data *C.uint8_t, length int) []byte {
 	return unsafe.Slice(uptr, length)
 }
 
+func value(handle C.uintptr_t) (value any, err error) {
```

Comment:
If we always use this to get a `*Client` can we make it return `(*Client, error)` and come up with a better name? (maybe `handleToClient()`)

Reply:
Makes sense, I've changed it in [082e2ff](https://github.com/gravitational/teleport/pull/32925/commits/082e2ff0a3dc000ae2a5839da22a3385a173dbdc)